actions/mongodb_query_admincommand_action_test.go

```diff
@@ -18,238 +18,296 @@ package actions
 import (
 	"context"
 	"io/ioutil"
-	"os"
 	"strings"
 	"testing"
 	"time"
 
 	"github.com/davecgh/go-spew/spew"
 	"github.com/percona/pmm/api/agentpb"
+	"github.com/pkg/errors"
 	"github.com/stretchr/objx"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/mongo"
 
-	"github.com/percona/pmm-agent/utils/templates"
 	"github.com/percona/pmm-agent/utils/tests"
 )
 
-func TestMongoDBBuildinfo(t *testing.T) {
+func TestMongoDBActions(t *testing.T) {
```

Comment:
looks like we could get rid of these duplicates just by using loop.

Reply:
as far as I see they are same, what is different?
I meant using loop for array with all DSNs.

Reply:
With one DSN used `getCmdLineOptsAssertionsWithAuth`, with another `getCmdLineOptsAssertionsWithSSL` and with another one `getCmdLineOptsAssertionsWithoutAuth`

Reply:
ok